sphinx_js/__init__.py

```diff
@@ -31,6 +34,7 @@ def setup(app):
     app.add_config_value('js_language', 'javascript', 'env')
     app.add_config_value('js_source_path', '../', 'env')
     app.add_config_value('jsdoc_config_path', None, 'env')
+    # app.add_config_value('jsdoc_cache', None, 'env')
```

Comment:
We should wipe this out before merging.

Reply:
You're right! I removed it in a new commit.
Maybe i committed that line in other prs too -.-